### PR TITLE
Fixed some bulding bugs in ubuntu

### PR DESCRIPTION
**Pull request by clockzhong**
Please merge it. Thanks!

### CODE REVIEW

File: include/mega/posix/megawaiter.h [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #define WAIT_CLASS PosixWaiter
 
 #include "mega/waiter.h"
-
+#include <mutex>
```
Comment (cb-mega) [inline include/mega/posix/megawaiter.h]: This is already included on `develop`. Is your fork out of date?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ Building
 
 For platforms with Autotools, the generic way to build and install it is:
 
+
+    sudo apt install libcrypto++-dev libsqlite3-dev libc-ares-dev libcurl4-openssl-dev zlib1g-dev libfreeimage-dev libreadline-dev libsodium-dev
```
Comment (cb-mega) [inline README.md]: The build dependencies are already listed further down under "Platform Dependencies".

File: include/mega/posix/megawaiter.h (hunk 1 above)
Comment (clockzhong) [reply include/mega/posix/megawaiter.h]: Yes, confirmed, this change was added several days ago, so we could close this issue.
.........
Commits on Oct 22, 2019
add missing include

@polmr
polmr committed 8 days ago

File: README.md (hunk 2 above)
Comment (clockzhong) [reply README.md]: Yes, it's my personal change to remind myself, I just want to copy&paste them when rebuilding the MegaSDK in a new Ubuntu env. So just ignore this. Thanks!

**Comment by clockzhong** [comment]
The following line:
 #include <mutex>
is already added into:
include/mega/posix/megawaiter.h

So close this request

**Pull request closed without merging**